decisions/0011-nmdc-edge-jetstream2.md

```diff
@@ -0,0 +1,36 @@
+---
+status: proposed
+date: 2024-04-27
+deciders: Mark Flynn, Shreyas Cholia
+consulted: Patrick Chain, Bin Hu
+informed: Emiley Eloe-Fadrosh
+---
+# New platform for hosting NMDC-EDGE
+
+## Context and Problem Statement
+Our plan is to move NMDC-EDGE from where it is currently hosted at SDSC. We would like to have a solution that is cheaper than our current deployment.
+
+## Considered Options
+
+* Google Cloud Projects (GCP)
+* Jetstream2 Virtual Clusters
```

Comment:
I recommend changing this bullet point into a hyperlink to https://docs.jetstream-cloud.org/general/virtualclusters/